packages/docusaurus/src/webpack/plugins/ChunkAssetPlugin.ts

```diff
@@ -25,16 +25,13 @@ class ChunkAssetPlugin {
           chunkNameToId[chunkName] = chunkId;
         });
         const buf = [source];
-        buf.push('');
         buf.push('// function to get chunk assets');
         buf.push(
           // If chunkName is passed, we convert it to chunk id
           // Note that jsonpScriptSrc is an internal webpack function
-          `${
-            mainTemplate.requireFn
-          }.gca = function(chunkId) { chunkId = ${JSON.stringify(
+          `__webpack_require__.gca = function(chunkId) { chunkId = ${JSON.stringify(
             chunkNameToId,
-          )}[chunkId]||chunkId; return jsonpScriptSrc(chunkId); };`,
+          )}[chunkId]||chunkId; return __webpack_require__.p + "" + (chunkId) + ".js"; };`,
```

Comment:
🤪 any doc link to share?

Reply:
This breaks the prefetch of chunks on link hover. Going to see how to fix this

![image](https://user-images.githubusercontent.com/749374/112993767-d79e4700-9169-11eb-900e-4dc78fcc00f4.png)